packages/components/src/components/Avatar/index.tsx

```diff
@@ -1,12 +1,18 @@
 import React from 'react';
 import css from '@styled-system/css';
-import { User } from '@codesandbox/common/lib/types';
 import styled from 'styled-components';
 import { Element } from '../Element';
 import { Text } from '../Text';
 
 interface IAvatarProps {
-  user: User;
+  user: {
+    id: string;
+    username: string;
+    name: string;
+    avatarUrl: string;
+    badges: any[];
+    subscriptionSince: string | null;
+  };
```

Comment:
Should we also remove fields that we don't use from this - like name and id

Reply:
it will give me an error saying it has too many fields passed through ahah

Reply:
@CompuIves  Is there a way to say I don't care about other fields? :P